CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Cara Berkontribusi
+
+## Panduan Mulai Bekerja
+
+Dalam proyek ini setiap ada _issues_ dan mengerjakan _issue_ silahkan gunakan Github agar semua aktivitas dapat terdokumentasi dan dapat diikuti kemajuan pengerjaannya dari waktu ke waktu.
+
+Sebelum mengerjakan sebuah _issue_, ada beberapa hal yang harus diperhatikan:
+
+1. Silahkan isi kolom *assignee* dengan username masing-masing.
+2. Untuk memulai proyek, pastikan **_create branch_** dari `main`, lalu segera *commit* dan *push* perubahannya sekecil apapun itu sebelum meninggalkannya. Kemudian **buat _pull request_** baru dengan menambahkan awalan `WIP:` di judulnya untuk menandainya sebagai _work in progress_. Dengan demikian, orang lain yang ingin mengerjakan _issue_ yang sama akan berpikir dua kali sebelum ikut mengerjakannya.
```

Comment:
This was the way to create Draft PRs in GitLab.
For GitHub, there's another way of creating Draft PRs, please [refer to this document](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) on how to do it, thanks!

Reply:
https://user-images.githubusercontent.com/6315466/124933073-2cd5d880-e036-11eb-922a-699014171a6c.mov